therapy/schemas.py

```diff
@@ -438,15 +439,142 @@ def schema_extra(schema: Dict[str, Any], model: Type["SourceMeta"]) -> None:
             }
 
 
-class NormalizationService(BaseModel):
-    """Response containing one or more merged records and source data."""
+class MatchesNormalized(BaseModel):
+    """Matches associated with normalized concept from a single source."""
+
+    records: List[Drug]
+    source_meta_: SourceMeta
+
+    class Config:
+        """Configure OpenAPI schema"""
+
+        @staticmethod
+        def schema_extra(schema: Dict[str, Any],
+                         model: Type["MatchesNormalized"]) -> None:
+            """Configure OpenAPI schema"""
+            if "title" in schema.keys():
+                schema.pop("title", None)
+            for prop in schema.get("properties", {}).values():
+                prop.pop("title", None)
+
+
+class BaseNormalizationService(BaseModel):
+    """Base method providing shared attributes to Normalization service classes."""
 
     query: str
     warnings: Optional[List[Dict]]
     match_type: MatchType
+    service_meta_: ServiceMeta
+
+
+class UnmergedNormalizationService(BaseNormalizationService):
+    """Response providing source records corresponding to normalization of user query.
+    Enables retrieval of normalized concept while retaining sourcing for accompanying
+    attributes.
+    """
+
+    normalized_concept_id: Optional[CURIE]
+    matches: Dict[SourceName, MatchesNormalized]
```

Comment:
`matches` --> `source_matches` to be consistent with `SearchService`?
```suggestion
    source_matches: Dict[SourceName, MatchesNormalized]
```